src/renderer/shared/core/types/balance.ts

```diff
@@ -20,3 +20,11 @@ export type BalanceLock = {
   type: LockTypes;
   amount: string;
 };
+
+export type TokenBalance = {
```

Comment:
I agree, would be better this way. I'd like to have there `BalanceDetails`or something like that and make it part of the `Balance` type. But struggle to come up with a meaningful name for it since it will be used separately as well 
maybe something like `BalanceSummary/BalanceDetails/BalanceAmount`?